### PR TITLE
The Shrimp Association Executive 

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Adds in a wellcheers aberration, Shrimp Association Executive.
How he works is that he requests a worktype, and then if you do that worktype, you get a work boost.
On good work, he gives you extra equipment for clerks to use lategame or maybe a spawn grenade to spawn helpers.
 

## Why It's Good For The Game
**This PR was requested by InsightfulParasite.**
He has been working very hard, and I want to make an abnormality that he helped design.

## Changelog
:cl:
add: Added The Shrimp Association Executive
/:cl:

### CODE REVIEW

**Review by dsmith328 — CHANGES_REQUESTED** [review]
Double Check your images and fix the EGO Armor from the console. Honestly, I don't think you'd need to manually add a weapon or at the very least you could have his "weapon" be the base Soda EGO gun, with the real ones gained via work. You could probably have the EGO Gift as a Tie as well.

**Comment by Kitsunemitsu** [comment]
Ego armor fixed, He's been given a golden soda pistol that costs 50 PE but is exactly the same in every other way.

File: code/datums/abnormality/_ego_datum/waw.dm [hunk 1]
```diff
@@ -70,12 +70,20 @@
...
 	cost = 25
-	
+
 // General Bee - Loyalty
 /datum/ego_datum/weapon/loyalty
 	item_path = /obj/item/gun/ego_gun/loyalty
 	cost = 50
 
 /datum/ego_datum/armor/loyalty
 	item_path = /obj/item/clothing/suit/armor/ego_gear/loyalty
```
Comment (dsmith328) [inline code/datums/abnormality/_ego_datum/waw.dm:80]: You deleted the cost for Loyalty here.

**Review by dsmith328 — CHANGES_REQUESTED** [review]
Besides this, most of it seems good

**Review by dsmith328 — APPROVED** [review]

**Pull request merged**